ConsoleApp/Program.cs

```diff
@@ -0,0 +1,490 @@
+﻿using System;
+
+using System.IO;
+
+using static System.Console;
+
+internal class Program
+{
+    const string filename = "data.csv";
+
+    static void Exit()
+    {
+        Environment.Exit(0);
+    }
+
+    static void ShowRow((string, string, string) row)
+    {
+        var (FirstName, LastName, Phone) = row;
+        WriteLine("{0,-15} {1,-15} {2,-15}", FirstName, LastName, Phone);
+    }
+
+    static void Show(string action, (string, string, string)[] result)
+    {
+        Clear();
+        WriteLine($"{action}");
+        WriteLine();
+        if (result.Length > 0)
+        {
+            ShowRow(("FirstName", "LastName", "Phone"));
+            foreach (var item in result)
+            {
+                ShowRow(item);
+            }
+            WriteLine("Press any key to continue...");
+            ReadKey();
+        }
+        else
+        {
+            Show("Search for a contact in the phone book.", "The contact not found.");
+            WriteLine();
+        }
+    }
+
+    static void Show(string action, string result)
+    {
+        Clear();
+        WriteLine($"{action}");
+        WriteLine();
+        WriteLine($"{result}");
+        WriteLine();
+        WriteLine("Press any key to continue...");
+        ReadKey();
+    }
+
+    static (string, string, string)[] ReadContacts(string file)
+    {
+        var contacts = new (string, string, string)[0];
+        if (File.Exists(file))
+        {
+            var lines = File.ReadAllLines(file);
+            contacts = new (string, string, string)[lines.Length];
+            for (int a = 0; a < lines.Length; ++a)
+            {
+                var items = lines[a].Split(",");
+                contacts[a] = (items[0], items[1], items[2]);
+            }
+        }
+        return contacts;
+    }
+
+    static bool WriteContacts(string file, string[] contacts)
+    {
+        if (File.Exists(file))
+        {
+            File.WriteAllLines(file, contacts);
+            return true;
+        }
+        else
```

Comment:
else is not needed here, please pay attention to it
you can simply `return false` after if branch